app/src/main/res/layout/fragment_movies_list.xml

```diff
@@ -0,0 +1,206 @@
+            android:layout_width="24dp"
+            android:layout_height="24dp"
+            android:layout_marginStart="6dp"
+            android:layout_marginTop="6dp"
+            android:alpha="0.9"
+            android:background="@drawable/age_limit_rectangle"
+            android:fontFamily="@font/roboto_black"
+            android:gravity="center"
+            android:text="@string/age_limit"
+            android:textColor="@color/white"
+            android:textSize="12sp"
+            app:layout_constraintStart_toStartOf="@id/movie_background_image_view"
+            app:layout_constraintTop_toTopOf="@id/movie_background_image_view" />
+
+        <ImageView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="6dp"
+            android:alpha="0.75"
+            android:src="@drawable/ic_like"
+            app:layout_constraintBottom_toBottomOf="@id/age_limit_text_view"
+            app:layout_constraintEnd_toEndOf="@id/movie_background_image_view"
+            app:layout_constraintTop_toTopOf="@id/age_limit_text_view" />
+
+        <TextView
+            android:id="@+id/film_genre"
+            android:layout_width="0dp"
+            android:layout_height="wrap_content"
+            android:layout_marginEnd="6dp"
+            android:layout_marginBottom="2dp"
+            android:fontFamily="@font/roboto_light"
+            android:text="@string/film_genre_drama"
+            android:textColor="@color/film_genre_color"
+            android:textSize="8sp"
+            app:layout_constraintBottom_toTopOf="@id/reviews"
+            app:layout_constraintEnd_toEndOf="@id/reviews"
+            app:layout_constraintStart_toStartOf="@id/first_star" />
+
+        <ImageView
+            android:id="@+id/first_star"
+            android:layout_width="8dp"
+            android:layout_height="8dp"
+            android:layout_marginStart="6dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintBottom_toBottomOf="@id/reviews"
+            app:layout_constraintStart_toStartOf="@id/movie_background_image_view"
+            app:layout_constraintTop_toTopOf="@id/reviews" />
+
+        <ImageView
+            android:id="@+id/second_star"
+            android:layout_width="8dp"
+            android:layout_height="8dp"
+            android:layout_marginStart="2dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintBottom_toBottomOf="@id/reviews"
+            app:layout_constraintStart_toEndOf="@id/first_star"
+            app:layout_constraintTop_toTopOf="@id/reviews" />
+
+        <ImageView
+            android:id="@+id/third_star"
+            android:layout_width="8dp"
+            android:layout_height="8dp"
+            android:layout_marginStart="2dp"
+            android:src="@drawable/ic_star_icon"
+            app:layout_constraintBottom_toBottomOf="@id/reviews"
```

Comment:
app:layout_constraintBottom_toBottomOf="@id/reviews"
app:layout_constraintTop_toTopOf="@id/reviews"

maybe look on stars in details layout?